third_party/terraform/utils/iam.go

```diff
@@ -199,8 +199,18 @@ func createIamBindingsMap(bindings []*cloudresourcemanager.Binding) map[string]m
 		}
 		// Get each member (user/principal) for the binding
 		for _, m := range b.Members {
+			// members are in <type>:<value> format
+			// <type> is case sensitive
+			// <value> isn't
+			// so let's lowercase the value and leave the type alone
+			pieces := strings.Split(m, ":")
```

Comment:
is it possible that the member could have a `:`  in it? If so, I think SplitN or something like that will make it so that you only split on the first occurrence.

Reply:
This is a good point! PR updated to use SplitN. Still look good?

Reply:
Yup!